features/configuration/run_all_when_everything_filtered.feature

```diff
@@ -1,78 +0,0 @@
-Feature: run all when everything filtered
```

Comment:
We still have the API docs.  I don't think it makes sense to highlight it in our relish docs.